send/send_lib.py

```diff
@@ -329,4 +329,22 @@ def exec_script(script_path: str, arguments: list[str]):
 	check_script_file(script_path)
 	command = [script_path]
 	command.extend(arguments)
-	return subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
\ No newline at end of file
+	return subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+
+
+def escape_filename(filename: str) -> str:
+	"""
+	Checks if provided filename contains any forbidden characters and escapes them.
+
+	:param filename: filename to escape
+	:return: escaped filename
+	"""
+	escaped_filename: str = ""
+	for char in filename:
```

Comment:
Wouldn't the 'replace' method be much simpler than a loop since we're escaping just one character?